src/core/Akka.Streams.Tests/Akka.Streams.Tests.csproj

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <AssemblyName>Akka.Streams.Tests</AssemblyName>
-    <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks>net452;netcoreapp1.1.1</TargetFrameworks>
```

Comment:
Testing a theory on the recent issues with .NET Core on Linux not being able to run specifications, based on what I read here: https://stackoverflow.com/a/43924403/377476